doc/tutorial/pairwise_alignment/configurations.cpp

```diff
@@ -69,10 +69,11 @@ auto sc_aa = aa_scheme.score('M'_aa27, 'K'_aa27); // sc_aa == 2.
 //! [gap_scheme]
```

Comment:
Maybe this should also be changed ?
```suggestion
//! [gap_cost_affine]
```